loco-new/base_template/src/tasks/user_delete.rs

```diff
@@ -0,0 +1,69 @@
+use crate::models::_entities::users;
+use loco_rs::prelude::*;
+use std::env;
+
+pub struct UserDelete;
+#[async_trait]
+impl Task for UserDelete {
+    fn task(&self) -> TaskInfo {
+        TaskInfo {
+            name: "user:delete".to_string(),
+            detail: "Delete a user by entering email.\nUsage:\ncargo run task user:delete"
+                .to_string(),
+        }
+    }
+    async fn run(&self, app_context: &AppContext, vars: &task::Vars) -> Result<()> {
+        let input = match vars.cli_arg("pid") {
+            Ok(email) => email,
+            Err(_) => return Err(Error::string("pid is mandatory")),
+        };
+
+        let user_to_delete = users::Model::find_by_pid(&app_context.db, &input).await?;
+
+        tracing::info!(
+            message = "User to delete",
+            user_email = &user_to_delete.email,
+            user_pid = user_to_delete.pid.to_string(),
+            "delete user via task"
+        );
```

Comment:
Instead of using `tracing::info!` here, I think we should just use `println!`.
If the log level is higher than `info`, the user would otherwise see the
confirmation prompt without any user details, which is not ideal for a
destructive action.